src/bin/generate_brackets.rs

```diff
@@ -1,16 +0,0 @@
-use nmg_league_bot::db::raw_diesel_cxn_from_env;
```

Comment:
this whole script was basically just a worse way of calling `/generate_pairings`